relstorage/storage.py

```diff
@@ -1073,6 +1073,9 @@ def _abort(self):
         if self.blobhelper is not None:
             self.blobhelper.abort()
 
+    def afterCompletion(self):
+        self._rollback_load_connection()
```

Comment:
A comment is less work, so attractive. :)

I'm not familiar with the internal state variables.  I was going to ask whether there should be a _store_transaction_open that this could check, but I'm puzzled by _load_transaction_open which is set but not checked by _rollback_load_connection. Shouldn't it be?